source/upgrade.txt

```diff
@@ -74,16 +74,43 @@ Detailed installation instructions may be found in the
 Breaking Changes
 ----------------
 
-A breaking change is a change of a convention or a behavior starting in a specific
-version of the library. This type of change may prevent your application from working
-properly if not addressed before upgrading the library.
+A breaking change is a change of a convention or a behavior starting in
+a specific version of the library. This type of change may prevent your
+application from working properly if not addressed before upgrading the
+library.
 
-The breaking changes in this section are categorized by the library version that introduced
-them. When upgrading library versions, address all the breaking changes between the current
-and upgrade versions.
+The breaking changes in this section are categorized by the library
+version that introduced them. When upgrading library versions, address
+all the breaking changes between the current and upgrade versions.
 
-For more information on release changes, see the release notes and associated 
-JIRA tickets for each release on `GitHub <https://github.com/mongodb/mongo-php-library/releases>`__.
+For more information on release changes, see the release notes and
+associated JIRA tickets for each release on `GitHub
+<https://github.com/mongodb/mongo-php-library/releases>`__.
+
+.. _php-v2-breaking-changes:
+
+Version 2.0 Breaking Changes
```

Comment:
I find it strange the `whats-new.txt` links to this page as a source of more information, but it actually contains less content. Is there any reason to have separate pages?

Reply:
We only use the upgrade guide to address **breaking changes**, whereas the whats new guide captures all changes. the upgrade guids can help users learn more about the specific steps they must take to address the breaking change instead of just describing the change